src/SFA.DAS.EmploymentCheck.Data.IntegrationTests/Repositories/EmploymentCheckCacheRequest/WhenSettingReleatedRequestsRequestCompletionStatus.cs

```diff
@@ -0,0 +1,188 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using FluentAssertions;
+using NUnit.Framework;
+using SFA.DAS.EmploymentCheck.Functions.Application.Enums;
+using SFA.DAS.EmploymentCheck.Functions.Repositories;
+using Models = SFA.DAS.EmploymentCheck.Functions.Application.Models;
+
+namespace SFA.DAS.EmploymentCheck.Data.IntegrationTests.Repositories.EmploymentCheckCacheRequest
+{
+    public class WhenSettingReleatedRequestsRequestCompletionStatus
+        : RepositoryTestBase
+    {
+        private IEmploymentCheckCacheRequestRepository _sut;
+        private IList<Functions.Application.Models.EmploymentCheckCacheRequest> _actual;
+
+        [Test]
+        public async Task CanSave()
+        {
+            // Arrange
+            _sut = new EmploymentCheckCacheRequestRepository(Settings);
+
+            var testEmploymentCheckCacheRequestData = await CreateTestEmploymentCheckCacheRequestData();
+            var expectedEmploymentCheckCacheRequestData = await CreateExpectedEmploymentCheckCacheRequestData();
+
+            // Act
+            foreach (var request in testEmploymentCheckCacheRequestData)
+            {
+                await base.Insert(request);
+            }
+
+            var result = new Tuple<Models.EmploymentCheckCacheRequest, ProcessingCompletionStatus>(testEmploymentCheckCacheRequestData.FirstOrDefault(), ProcessingCompletionStatus.Skipped);
+            await _sut.SetReleatedRequestsRequestCompletionStatus(result);
+
+            // Assert
+            _actual = (await GetAll<Functions.Application.Models.EmploymentCheckCacheRequest>()).OrderBy(x => x.Id).ToList();
+
+            for (var i = 1; i < _actual.Count; i++)
+            {
+                var expected = expectedEmploymentCheckCacheRequestData[i];
+
+                _actual[i].Should().BeEquivalentTo(expected,
+                    opts => opts
+                        .Excluding(x => x.Id)
```

Comment:
think Id can actually be compared from what we found out the other day. remove `.Excluding(x => x.Id)` and `_actual[i].Id.Should().BeGreaterThan(0);` below.